packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -236,13 +236,42 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types, is_dark_mod
                     </Text>
                 </Button>
             )}
-            <ActionSheet.Root isOpen={is_open} expandable={false} onClose={handleCloseTradeTypes}>
+            <ActionSheet.Root
+                className='trade-types-dialog'
+                isOpen={is_open}
+                expandable={false}
+                onClose={handleCloseTradeTypes}
+            >
                 <ActionSheet.Portal shouldCloseOnDrag>
                     <ActionSheet.Header
-                        title={<Localize i18n_default_text='Trade types' />}
+                        title={
+                            <div className='trade-types-dialog__title'>
```

Comment:
Maybe we can just pass `trade-types-dialog__title` as `className` prop to `ActionSheet.Header`?